CICD.md

```diff
@@ -0,0 +1,12 @@
+### Deploy to Heroku
+1. Login to Heroku by running `heroku login`
+2. Create a new Heroku application and push your application to a Git remote repository
+    1. Run `heroku create` 
```

Comment:
probably want to add that you should specify a name or heroku will generate a random one for you